assets/app/scripts/filters/util.js

```diff
@@ -117,6 +117,9 @@ angular.module('openshiftConsole')
           }
           unit += (amount === "1" ? "core" : "cores");
           break;
+        case "storage":
+          unit = unit.substring(0, 1);
```

Comment:
Sometimes it's just `G` and sometimes it's `Gi` as referred by [some docs](https://docs.openshift.org/latest/dev_guide/persistent_volumes.html#requesting-storage). I'll keep substring-ing it but I'll add `B` like memory does, so that we have `GB`.